mtp_noms_ops/templates/security/accept_or_reject_check.html

```diff
@@ -18,6 +18,7 @@ <h1 class="heading-xlarge">{{ view.title }}</h1>
     {% include 'mtp_common/forms/error-summary.html' with form=form only %}
 
     <h2 class="heading-medium">{% trans 'Credit details' %}</h2>
+    <div class="mtp-results-list table-font-xsmall mtp-check-list">
     <table class="mtp-table mtp-table--review">
```

Comment:
Is it worth indenting the table now there's a surrounding div, really minor point of course!

Reply:
yes i totally would! i just didn't want to change every line and have this not be mergeable with #479. actually i checked this morning and the SCSS file doesn't merge. so i don't mind this PR going on top of the other once that's merged. or vice versa if you like?

Reply:
My strong preference would be that this PR gets merged after #479